Sources/NIO/SocketAddresses.swift

```diff
@@ -112,12 +112,12 @@ public enum SocketAddress: CustomStringConvertible {
     }
 
     /// Get the port associated with the address, if defined.
-    public var port: UInt16? {
+    public var port: Int? {
```

Comment:
@weissi isn't that a public API change and so breaking ?

Reply:
@normanmaurer yes, that's why it's marked as "⚠️ needs-major-version-bump" and "[2.0]" :).

Just wanted to add this PR (to merge when we decide to release 2.0) just so we won't forget this change.